From 24340049229343f3fc078a19415d095e97d797e6 Mon Sep 17 00:00:00 2001
From: Tanu Kaskinen <tanu.kaskinen@linux.intel.com>
Date: Tue, 20 Jan 2015 15:36:44 +0100
Subject: [PATCH 5/6] combine-sink: Fix the initial requested latency of new
 outputs

If the combine-sink is running with low latency, and a new output is
added, the new output was not configured with low latency.
---
 src/modules/module-combine.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/module-combine.c b/src/modules/module-combine.c
index bac333a..326674d 100644
--- a/src/modules/module-combine.c
+++ b/src/modules/module-combine.c
@@ -941,7 +941,7 @@ static int output_create_sink_input(struct output *o) {
     o->sink_input->kill = sink_input_kill_cb;
     o->sink_input->userdata = o;
 
-    pa_sink_input_set_requested_latency(o->sink_input, BLOCK_USEC);
+    pa_sink_input_set_requested_latency(o->sink_input, pa_sink_get_requested_latency(u->sink));
 
     return 0;
 }
-- 
1.9.3

